### ISSUE TITLE
Can't disasm Thumb2 instruction? D0 F8 58 90 for instance

### ISSUE THREAD

**Issue by orlando1986** [cea-sec/miasm#686]
miasm seems can't disam arm thumb2 instruction, for example:
D0 F8 58 90
in IDA, it represents:
LDR.W           R9, [R0,#0x58]
but miasm return nothing

**Comment by serpilliere** [comment]
Yes: there are a lot of thumb "T3" encoded style instructions which are not supported yet.
I May do a PR in the future which includes the missing instructions
